ipykernel/eventloops.py

```diff
@@ -125,7 +125,8 @@ def loop_qt4(kernel):
 @register_integration('qt', 'qt5')
 def loop_qt5(kernel):
     """Start a kernel with PyQt5 event loop integration."""
-    os.environ['QT_API'] = 'pyqt5'
+    if os.environ['QT_API'] not in ['pyqt5', 'pyside2']:
+        os.environ['QT_API'] = 'pyqt5'
```

Comment:
I really don't understand this change. I mean, this

    if os.environ['QT_API']

assumes that `QT_API` is already in the environment, which is not always the case.

And this

    not in ['pyqt5', 'pyside2']

seems to assume that the only possible values of `QT_API` can be `pyqt5` and `pyside2`, when they can also be `pyqt` and `pyside`.